torch_frame/gbdt/tuned_xgboost.py

```diff
@@ -225,7 +235,11 @@ def _predict(self, tf_test: TensorFrame) -> Tensor:
         dtest = xgboost.DMatrix(test_feat, label=test_y,
                                 feature_types=test_feat_type,
                                 enable_categorical=True)
-        pred = self.model.predict(dtest)
+        if self.model.best_iteration is not None:
+            iteration_range = self.model.best_iteration
+        else:
+            iteration_range = None
```

Comment:
ditto

Reply:
It's possible to hit this case, when there's no early stopping